conf/map/battle/feature.conf

```diff
@@ -98,4 +98,9 @@ features: {
 	// true:  enable
 	// false: disable (default)
 	replace_refine_npcs: false
+
+	// Enable Multi-Critical damage
+	// true:  enable (Default)
+	// false: disable
```

Comment:
```suggestion

```
perhaps the comment aren't needed? its pretty much self-explained.

Reply:
ACK